lib/metasploit/framework/password_crackers/jtr/formatter.rb

```diff
@@ -57,6 +57,10 @@ def hash_to_jtr(cred)
       # https://moar.so/blog/qnx-password-hash-formats.html
       hash = cred.private.data.end_with?(':0:0') ? cred.private.data : "#{cred.private.data}:0:0"
       return "#{cred.public.username}:#{hash}"
+    when /Raw-MD5/
+      # not md5-crypt, just an md5. Avira uses this with unicode
+      # the trailing : shows an empty salt, JTR and hashcat compatible
+      return "#{cred.private.data}:"
```

Comment:
those fields would only potentially be needed if it were part of a `aux/analyze` module.  Since this hash is such a one off, I don't see it getting added to any of the cracker modules.  With this in mind, its ok as is.  However, if a future expansion of the cracker modules added this, it would have to be evaluated at that time to see how jtr/hashcat handled it and deconflicted.

Reply:
Ok sounds good